scipy/stats/tests/test_continuous_fit_censored.py

```diff
@@ -529,7 +529,9 @@ def test_nct():
     data = CensoredData.right_censored([1, 2, 3, 5, 8, 10, 25, 25],
                                        [0, 0, 0, 0, 0, 0, 1, 1])
     # Fit just the shape parameter df and nc; loc and scale are fixed.
-    df, nc, loc, scale = nct.fit(data, floc=0, fscale=1, optimizer=optimizer)
+    with np.errstate(over='ignore'):  # remove context once gh-14901 is resolved
+        df, nc, loc, scale = nct.fit(data, floc=0, fscale=1,
+                                     optimizer=optimizer)
```

Comment:
Hopefull will silence
```
FAILED scipy/stats/tests/test_continuous_fit_censored.py::test_nct - RuntimeWarning: overflow encountered in _nct_sf
FAILED scipy/stats/tests/test_continuous_fit_censored.py::test_ncx2 - RuntimeWarning: overflow encountered in _ncx2_sf
```
This is a known issue tracked in gh-14901. I'll mention there that this can be removed as part of resolving that issue.